tests/IntlFormatterTest.php

```diff
@@ -174,6 +168,22 @@ public function testIssue6($tokens_values, $expect)
         $this->assertSame($expect, $actual);
     }
 
+    public function testEmptyStringThrowsException()
+    {
+        $locale = 'en_US';
+        $string = '';
+        $formatter = $this->newFormatter();
+        if (defined('PHP_VERSION')) {
+            $this->setExpectedException('Aura\Intl\Exception\CannotInstantiateFormatter');
+        }
+        $actual = $formatter->format($locale, $string, []);
+        if (defined('HHVM_VERSION')) {
```

Comment:
Hi @lorenzo , @pmjones and anyone who is looking into this, 

Can you give me some feedback on this particular test case ?

The reason is if we pass empty string php cannot instantiate MessageFormatter, but HHVM can. So the behaviour of this test is a bit different. Do you think we need to check if it is HHVM and thow the same exception at the source level rather than fixing this in the test ?

Please see https://3v4l.org/Y2pZ2 .